dyn_em/module_diffusion_em.F

```diff
@@ -8146,15 +8160,21 @@ SUBROUTINE vertical_diffusion_implicit(ru_tendf, rv_tendf, rw_tendf, rt_tendf,&
     DO  i = i_start, i_end
         DO  k = kts, ktf-1
            rdz_w = - g/dnw(k)/(c1h(k)*mu(i,j)+c2h(k))
-           beta  =   1.5*sqrt(tke(i,k,j))/l_diss(i,k,j)
+           IF (l_diss(i,k,j) .ne. 0) THEN
+               beta  =   1.5*sqrt(tke(i,k,j))/l_diss(i,k,j)
+           ELSE
+               beta  =   0.
+           ENDIF
```

Comment:
l_diss can be zero if hpbl=0 and dlk=0 (resulting from tke=0). Not sure whether setting beta=0 is appropriate here.

Reply:
beta indicates the dissipation of TKE. When the length scale of dissipation (l_diss) is zero, the dissipation is zero. I agree with the fix.

Reply:
I agree with the fix.